.github/CODEOWNERS

```
@@ -0,0 +1,2 @@
+* @GeorgeJahad @harageth @itzg @jjbuchan @rohitsngh27 @shabd67
```

Oops, I shouldn't have mentioned in Slack, I had created a team `salus-reviewers` to see if this was going to work instead of listing us individually:
```suggestion
* @racker/salus-reviewers
```

---

If it works, it'll be an easy change now the PRs are open.  I just opened all repos in iterm and created the files and pushed the commits across all in one go.